src/s3a_the_gutter.tw2

```diff
@@ -1,174 +1,913 @@
+"What up Donnie?!" I screamed across the bar and raised the horns.
+
+Donnie raised them back and we [[exchanged a righteous nod|The Gutter - Inside hub]]. That kid was alright.
+
+::The Gutter - Bathroom
+Fuck this place, fuck this smoke, fuck these people.
+
+I tromped through the bar's blue clouds and gave the door to the 'Lettes room two sharp raps before barreling in. With a crowd like the one out there, it was pretty unlikely that anyone was inside.
+
+And yup — as I turned the handle and swung the door, the bathroom stood empty. I closed the door tight behind me and kicked the stopper in under the bottom edge.
+
+Next best thing to a lock if you were cheap as fuck. And anyway, bartender still needed to be able to make sure ninjas weren't getting wet and wild on the fucking shitter.
+
+(click: "wet and wild on the fucking shitter")[As soon as I turned away from the door, I (live: 1s)[(if: time < 20s)[$dissolve[[[started puking into the toilet|The Gutter - Bathroom - Toilet puke]]]]
+\(else-if: time > 20s)[$dissolve[[[started puking on myself|The Gutter - Bathroom - Sink puke]](stop:)]]].]
+
+(set: $mirrorSeen to false)(set: $toiletflashbackSeen to false)(set: $bathroomVisited to true)
+
+::The Gutter - Bathroom - Toilet puke
+I barely propped the toilet lid open fast enough to hit the mark. I barfed a strong, hateful stream into the bowl.
+
+When I was finished I was gasping, like I’d been punched in the gut. This night was all too much. Anna gone, maybe dead already. I’ve been drugged, chased, and now I’ve gotta ask my questions with puke on my breath.
+
+I wiped my mouth with the back of my hand. When this was over, I was gonna need a high the length of I-55 to deal.
+
+(display: "The Gutter - Bathroom - Menu")
+
+::The Gutter - Bathroom - Sink puke
+I spewed all over the floor, splashing on my feet, before I finally found the sink. I nearly filled it up with whatever needed out of me.
+
+When I was finished I was gasping, like I’d been punched in the gut. This night was all too much. Anna gone, maybe dead already. I’ve been drugged, chased, and now I’ve gotta ask my questions with puke on my shoes.
+
+I wiped my mouth with the back of my hand. When this was over, I was gonna need a high the length of I-55 to deal.
+
+(display: "The Gutter - Bathroom - Menu")
+
+(set: $sinkPuke to true)
+
+::The Gutter - Bathroom - Menu
+>>[[I headed back out into the din and dipshits|The Gutter - Inside hub]]
+(if: $mirrorSeen is false)[>>[[I was drawn to my reflection in the mirror|The Gutter - Bathroom - Mirror]].]
+(if: $toiletflashbackSeen is false)[>>[[I sat on the toilet to catch my breath|The Gutter - Bathroom - Toilet flashback]].]
+
+::The Gutter - Bathroom - Mirror
+(set: $mirrorSeen to true)
+I gripped the edges of the counter a bit tighter. My face looked unfamiliar. The stranger’s eyes were weary, like they’d been hung with weights.
+
+//“Hey. $playerName," the mirror stranger said, "You look pretty fucked up. Like somebody dosed you.”//
```

Comment:
The `Hey. $playerName,` looks weird with a period after "Hey" — what about:

`"Hey, $playerName," the mirror stranger said. "You look pretty fucked up. Like somebody dosed you."`
(Note the added period after "said.")

Reply:
Also, I vote that we not italicize "the mirror stranger said." Further down in this sequence you stop italicizing the description between quotes from the mirror stranger, and I think that reads better.

Reply:
Without the italics, it's pretty confusing to tell who's talking to who. Since it's not working, I removed it and just added more text to signify who's talking when.